test/standalone_gutters_test.cpp

```diff
@@ -0,0 +1,329 @@
+#include <gtest/gtest.h>
```

Comment:
This file should not exist, move any desired tests to the guttering_systems_test.ccp file